broadinstitute/pearl
# AR-222 more OurHealth Basic survey touchups

This adds a state postal code dropdown (we'll want to refactor this out into a common component later I'm sure), changes a radiogroup to a checkbox, and adds "(5 digits)" to the zip code title.

TO TEST:
1. repopulate ourhealth
2. sign in to participant view as consented@test.com
3. take basic survey, confirm postal code dropdown appears, and also the qualifier text on "zip code" title
